debugwidget.py

```diff
@@ -388,10 +388,9 @@ def current_frame_changed(self, current, previous):
     def go_to_frame(self, index):
 
         filename = self.entries[index][0]
-        lineno = self.entries[index][1]
+        self.source.line = self.entries[index][1]
 
         self.source.openFile(filename)
-        self.source.jumpToLine(lineno)
```

Comment:
Does this break changing between the frames via the panel on the left?

Reply:
Good point, it does.